README.md

```diff
@@ -4,6 +4,9 @@
   <p align="center">Build, develop, and deploy Next.js apps for Cloudflare Pages.</p>
 </p>
 
+> [!NOTE]  
+> The best way to run Next.js apps on Cloudflare is to use [`@opennextjs/cloudflare`](https://opennext.js.org/cloudflare). If you're coming from Vercel, you can migrate your Next.js app easily using [Diverce](https://github.com/ygwyg/diverce), which can also help you migrate from `cloudflare/next-on-pages`
```

Comment:
```suggestion
> The best way to run Next.js apps on Cloudflare is to use [`@opennextjs/cloudflare`](https://opennext.js.org/cloudflare). If you're coming from Vercel, you can migrate your Next.js app using a community tool, [Diverce](https://github.com/ygwyg/diverce), which can also help you migrate from `cloudflare/next-on-pages`
```

Reply:
the last `cloudflare/next-on-pages` should have an `@` symbol as well :)

Reply:
```suggestion
> The best way to run Next.js apps on Cloudflare is to use [`@opennextjs/cloudflare`](https://opennext.js.org/cloudflare). If you're coming from Vercel, you can migrate your Next.js app easily using [Diverce](https://github.com/ygwyg/diverce), which can also help you migrate from `@cloudflare/next-on-pages`
```